bluesky/utils.py

```diff
@@ -1536,25 +1536,51 @@ def _rearrange_into_parallel_dicts(readings):
     return data, timestamps
 
 
+def is_movable(obj):
+    """Check if object satisfies bluesky 'movable' interface.
+
+    Parameters
+    ----------
+    obj : Object
+        Object to test.
+
+    Returns
+    -------
+    boolean
+        True if movable, False otherwise.
+    """
+    EXPECTED_ATTRS = (
+        'name',
+        'parent',
+        'read',
+        'describe',
+        'read_configuration',
+        'describe_configuration',
+        'set',
+        'stop',
+    )
+    return all(hasattr(obj, attr) for attr in EXPECTED_ATTRS)
+
+
 class Movable(metaclass=abc.ABCMeta):
     """
     Abstract base class for objects that satisfy the bluesky 'movable' interface.
-
     Examples
     --------
-
     .. code-block:: python
-
```

Comment:
Looks like this change may break proper rendering by Sphinx. Can it be reverted (and the removed lines above)? Thanks.

Reply:
I pushed a commit with inserted strings. I guess once CI passes we can merge it.